vsts/pipelines/buildImageBases.yml

```diff
@@ -10,41 +10,41 @@ stages:
   - template: templates/_buildimageBasesJobTemplate.yml
     parameters:
       displayName: Build and push Node base images for runtime
-      scriptPath: ./build/build-runtimeimages-bases.sh
+      scriptPath: ./build/buildRunTimeImageBases.sh
       imageDir: node
       artifactsFileName: node-runtimeimage-bases.txt
 
   - template: templates/_buildimageBasesJobTemplate.yml
     parameters:
       displayName: Build and push Python base images for build
-      scriptPath: ./build/build-buildimage-bases.sh
+      scriptPath: ./build/buildBuildImageBases.sh
       imageDir: python
       artifactsFileName: python-buildimage-bases.txt
 
   - template: templates/_buildimageBasesJobTemplate.yml
     parameters:
       displayName: Build and push PHP build base images
-      scriptPath: ./build/build-buildimage-bases.sh
+      scriptPath: ./build/buildBuildImageBases.sh
       imageDir: php
       artifactsFileName: php-buildimage-bases.txt
 
   - template: templates/_buildimageBasesJobTemplate.yml
     parameters:
       displayName: Build and push PHP runtime base images
-      scriptPath: ./build/build-runtimeimages-bases.sh
+      scriptPath: ./build/buildRunTimeImageBases.sh
       imageDir: php
       artifactsFileName: php-runtimeimage-bases.txt
 
   - template: templates/_buildimageBasesJobTemplate.yml
     parameters:
       displayName: Build and push Yarn Cache base image
-      scriptPath: ./build/build-buildimage-bases.sh
+      scriptPath: ./build/buildBuildImageBases.sh
       imageDir: yarn-cache
       artifactsFileName: yarn-cache-buildimage-bases.txt
 
 - stage: Release
   displayName: Release Stage
-  dependsOn: Build
+  condition: or(succeeded(), failed())
```

Comment:
this will make the release stage to run always, but it will wait for the build stage to finish first